frontend/src/components/dialogs/GDialog.vue

```diff
@@ -202,6 +202,23 @@ export default {
         resolve(value)
       }
       this.visible = false
+    },
+    showScrollBar () {
+      const contentCardRef = this.$refs.contentCard
+      if (!contentCardRef || !contentCardRef.clientHeight) {
+        this.$nextTick(() => this.showScrollBar())
+        return
+      }
+      const scrollTopVal = contentCardRef.scrollTop
+      contentCardRef.scrollTop = scrollTopVal + 10
+      contentCardRef.scrollTop = scrollTopVal - 10
+    }
+  },
+  watch: {
+    visible (value) {
+      if (value) {
+        this.showScrollBar()
```

Comment:
why not call this.showScrollBar() surrounded with nextTick here instead of having the nextTick within the showScrollBar.
If you want to keep it in the showScrollBar then an abort condition would be good by passing either a boolean or a counter to the showScrollBar function

Reply:
The reason ist that one next tick is sometimes not sufficient, it takes some time until all refs are set.

Reply:
then we should use a counter/tries parameter

Reply:
done